app/Helpers/ResponseBuilder.php

```diff
@@ -14,7 +14,7 @@ class ResponseBuilder implements ResponseInterface
      * @param string $error
      * @return \Illuminate\Http\JsonResponse|object
      */
-    public function apiJsonResponse($code, $message, $data, $error)
+    public function apiJsonResponse($code, $message, $data, $error = null)
```

Comment:
This is incorrect. Please remove `=null`. As it is not the correct implementation of the interface.
```javascript
public function apiJsonResponse($code, $message, $data, $error);
```

Reply:
Done